### PR TITLE
Update grpc notes

### PR DESCRIPTION
**Pull request by rafwegv**
### Proposed changes
Update note about grpc unary support in documents

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto master
- [x] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

**Comment by Dean-Coakley** [comment]
Seems like there is also links that 404 on https://deploy-preview-1468--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/app-protect/configuration/#app-protect-logs ?

File: docs-web/app-protect/configuration.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ You can define App Protect policies for your Ingress resources by creating an `A
 
  > **Note**: The fields `policy.signature-requirements[].minRevisionDatetime` and `policy.signature-requirements[].maxRevisionDatetime` are not currently supported. 
 
- > **Note**: [The Advanced gRPC Protection for Unary Traffic](/nginx-app-protect/configuration/#advanced-grpc-protection-for-unary-traffic) is not currently supported.
+ > **Note**: [The Advanced gRPC Protection for Unary Traffic](/nginx-app-protect/configuration/#advanced-grpc-protection-for-unary-traffic) only supports providing an `idl-file` inline. The fields `policy.idl-files[].link`, `policy.idl-files[].$ref`, and 
+ `policy.idl-files[].file` are not supported. The IDL file should be provided in field `policy.idl-files[].contents`. The value of this field can be base64 encoded. In this case the field `policy.idl-files[].isBase64` should be set to `true`.
```
Comment (pleshakov) [inline docs-web/app-protect/configuration.md:22]:
```suggestion
 `policy.idl-files[].file` are not supported. The IDL file should be provided in the field `policy.idl-files[].contents`. The value of this field can be base64 encoded. In this case the field `policy.idl-files[].isBase64` should be set to `true`.
 ```

**Review by pleshakov — APPROVED** [review]

**Comment by pleshakov** [comment]
@rafwegv is this PR still WIP?

**Comment by rafwegv** [comment]
@pleshakov I kept it as WIP until the updated notes became true (grpc support was merged:) )

**Review by lucacome — APPROVED** [review]

**Pull request merged**